plugins/modules/compute_resource.py

```diff
@@ -360,6 +362,7 @@ def main():
                 tenant=dict(),
                 app_ident=dict(),
                 datacenter=dict(),
+                caching_enabled=dict(type='bool'),
```

Comment:
You'll have to add that to the DOCUMENTATION string in the suboptions of `provider_params`:
https://github.com/theforeman/foreman-ansible-modules/blob/15db065de93584b561aec4bee3990aaf3341ac23/plugins/modules/compute_resource.py#L54-L58

See `use_v4` or `ssl_verify_peer` for examples how to do this :)

Reply:
Added